specs/hash.table

```diff
@@ -10,6 +10,10 @@ schema([
 extended_schema(POSIX, [
     Column("ssdeep", TEXT, "ssdeep hash of provided filesystem data"),
 ])
+extended_schema(LINUX, [
+    Column("pid_with_namespace", INTEGER, "Pids that contain a namespace", additional=True),
```

Comment:
Nitpick: Is there perhaps an alternate column name we could use? I do not like having a preposition in the name.

Reply:
I'm not sure honestly, meaning the column definition is that "this is a pid (or pids) that contain(s) the namespace you want to query.

`namespace_pid` is the closest thing although a namespace doesn't have a pid...

I was going for something that could be read as english while reading the SQL query.

Reply:
It looks like a lot of work has gone into development and reviewing the code. I suggest landing and then we can follow up if needed.

Aside: I've wanted to experiment with an API like:

```
select * from file(pid_with_namespace=14677) WHERE path LIKE "/%"
```

I am not sure how to implement that "table used like a function" behavior, but for containers it feels like the right way to abstract out how to select the container(s) to act on.

Reply:
Something like this maybe https://www.sqlite.org/vtab.html#tabfunc2?

Mind, the column still exists because it has to be passed as a constraint, but it's hidden and not passed as a column in the query.

Reply:
My initial thought is that `file(pid_with_namespace=14677) ` looks appealing, but I'm not sure why we'd treat it different than a hidden required column. 
```
select * from file WHERE pid_with_namespace=14677 and path LIKE "/%"
```

Sorta thinking aloud, since we're using this sql model, we have columns that convey options and arguments, and columns that return data. If we opted to treat them separately:
* `select * from file WHERE pid_with_namespace=14677 and path LIKE "/%"`
* `select * from file(pid_with_namespace=14677) WHERE path LIKE "/%"`
* `select * from file(pid_with_namespace=14677, path="/%")`

Initially, I feel like forcing everything through a column creates a consistency. But I probably need to think more